.github/workflows/build.yml

```diff
@@ -57,7 +57,11 @@ jobs:
         working-directory: ./build
         run: cmake --build .
       - name: Report build details
+        if: success() || failure()
         run: du -hbcs ./source ./build
+      - name: Run tests
+        working-directory: ./build
+        run: ctest -C ${{matrix.build-type}} -LE REQUIRES_HARDWARE
```

Comment:
I added this even for android. Right now it runs 0 tests as they are all set to require hardware. This is here justs in case in the future we want to have host only tests or something like that